src/components/Filters/__tests__/Company.spec.jsx

```diff
@@ -3,9 +3,9 @@ import React from 'react'
 import { IntlProvider } from 'react-intl'
 import { Provider } from 'react-redux'
 import renderer from 'react-test-renderer'
+import { Company } from '../Company'
 import configureMockStore from 'redux-mock-store'
 import thunk from 'redux-thunk'
-import ReduxCompany, { mapDispatchToProps, Company } from '../Company'
```

Comment:
I'm going to thread these comments because they belong together. 🍒   vs 🍓 

Notice this declaration is: default = `ReduxCompany` + 2 exports

Reply:
whoops, i put it back.